backend/src/main/java/se/chalmers/dat265/group1/model/dbEntities/OrderData.java

```diff
@@ -0,0 +1,38 @@
+package se.chalmers.dat265.group1.model.dbEntities;
+
+import se.chalmers.dat265.group1.model.DataModel;
+import se.chalmers.dat265.group1.model.Order;
+
+import java.util.LinkedList;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.LinkedList'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)